File: 코딩 기초 트레이닝/배열의 길이를 2의 거듭제곱으로 만들기-p181857/배열의 길이를 2의 거듭제곱으로 만들기.py
# https://school.programmers.co.kr/learn/courses/30/lessons/181857

def solution(arr):
    lens = len(arr)

    if(lens & (lens-1) == 0):
        return arr
    else:
        while(lens & (lens-1) != 0):
            arr.append(0)
            lens = len(arr)
    return arr

File: 코딩 기초 트레이닝/배열의 길이를 2의 거듭제곱으로 만들기-p181857/배열의 길이를 2의 거듭제곱으로 만들기.js
// https://school.programmers.co.kr/learn/courses/30/lessons/181857

function solution(arr) {

    if(arr.length & arr.length -1 === 0) {
        return arr;
    }

    while((arr.length & arr.length -1 )!== 0) {
        arr.push(0);
    }

    return arr;
}
